chirp/wxui/developer.py

```diff
@@ -347,7 +347,7 @@ def selected(self):
             label = wx.StaticText(self)
             pos = wx.StaticText(self, label='%i bits (%i bytes) at 0x%06x' % (
                     self._memobj.size(),
-                    self._memobj.size() // 8,
+                    int(self._memobj.size() // 8),
```

Comment:
Thanks. I've been meaning to track this down. However, this just papers over the issue, because `//` should return an integer of course, and not need to be int-ified like this. I'll push a fix separately and you can drop this.